src/main/java/uk/gov/hmcts/darts/common/entity/EventEntity.java

```diff
@@ -44,8 +45,8 @@ public class EventEntity {
     @Column(name = "c_time_stamp")
     private OffsetDateTime timestamp;
 
-    @Column(name = "c_courtroom")
-    private String courtroom;
+    @ManyToOne(fetch = FetchType.LAZY)
+    private CourtroomEntity courtroom;
```

Comment:
i could be wrong, but i thought that events were linked to a hearing, and it was the hearing that told us what courthouse, courtroom and case number it was for. I don't see case_number here either.

Reply:
If true, barrys script hasn't been updated.  That was my assumption at first also.